totp/cli.py

```diff
@@ -111,6 +111,27 @@ def add_uri(path, uri):
     totp.add_pass_entry_from_uri(path, uri)
 
 
+@subcommand(
+    "rm",
+    argument(
+        "identifier",
+        help="the identifier under the '2fa' folder where the key to remove is saved",
+    ),
+    aliases=["-r"],
+    description="Remove a TOTP entry from the database.",
+    help="remove a TOTP entry from the database",
+)
+def _cmd_rm(args):
+    if args.identifier:
```

Comment:
I think it's a good idea to add a confirmation before deleting:

https://click.palletsprojects.com/en/8.1.x/prompts/#confirmation-prompts

I think that is might be feasible to use add a `--force` option which prompts for manual confirmation if not specified: https://click.palletsprojects.com/en/8.1.x/options/#option-prompting

This might also be a good fit: https://click.palletsprojects.com/en/8.1.x/api/#click.confirmation_option

Reply:
I think it's an excellent idea to add a confirmation. Especially in the situation of TOTP... I unfortunately have little time. I'll see if I come around to this.

Reply:
OK. Added only the confirmation prompt. (For now at least).

There is now one extra dependency on `click`. Maybe time to unify the command line parsing with `click` instead of `argparse` ? Or is `click` overkill ?